Makefile

```diff
@@ -88,12 +88,12 @@ endif
 
 .PHONY: run-e2e-tests
 run-e2e-tests:
-	KUBECONFIG=${kubeconfig} go test -mod=$(MOD) -tags="$(platform),e2e" -covermode=atomic -buildmode=exe -v ./...
+	KUBECONFIG=${kubeconfig} go test -timeout=20m -mod=$(MOD) -tags="$(platform),e2e" -covermode=atomic -buildmode=exe -v ./...
```

Comment:
Why increase time test?

Reply:
Good question. BTW default Go timeout for tests is 10m.

Reply:
Yeah I increased the timeout because some tests took little longer in some cases leaving no time for other tests to pass. Increasing the timeout helped me get rid of that problem.